### ISSUE TITLE
Redgifs broken

### ISSUE THREAD

**Issue by thickconfusion** [stashapp/CommunityScrapers#1809]
** Scraper name **
Redgifs

** Scraper method **
Scene

```
ERRO[2024-04-26 19:54:50] [Scrape / Redgifs] HTTP Error: 404
ERRO[2024-04-26 19:54:50] [Scrape / Redgifs] Traceback (most recent call last):
ERRO[2024-04-26 19:54:50] [Scrape / Redgifs] File "/root/.stash/scrapers/community/Redgifs/Redgifs.py", line 183, in
ERRO[2024-04-26 19:54:50] [Scrape / Redgifs] result = json.dumps([scraper.getParseId(id)])
ERRO[2024-04-26 19:54:50] [Scrape / Redgifs] ^^^^^^^^^^^^^^^^^^^^^^
ERRO[2024-04-26 19:54:50] [Scrape / Redgifs] File "/root/.stash/scrapers/community/Redgifs/Redgifs.py", line 124, in getParseId
ERRO[2024-04-26 19:54:50] [Scrape / Redgifs] gif = req.get("gif")
ERRO[2024-04-26 19:54:50] [Scrape / Redgifs] ^^^^^^^
ERRO[2024-04-26 19:54:50] [Scrape / Redgifs] AttributeError: 'NoneType' object has no attribute 'get'
ERRO[2024-04-26 19:54:50] could not unmarshal json from script output: EOF
ERRO[2024-04-26 19:54:50] scrapeSingleScene: input: scrapeSingleScene error while name scraping with scraper Redgifs: could not unmarshal json from script output: EOF
```

**Comment by Maista6969** [comment]
Please update your Redgifs scraper to the latest version in the Community feed, `afef303`, and report back if the error still occurs

**Comment by thickconfusion** [comment]
Updated and it shows `afef303` installed. No more error, but no results found on several scenes. Could be my misunderstanding of running it.

**Comment by Maista6969** [comment]
When using "Scrape with..." it will try to find the GIF identifier (something like `navajowhitepristineamericanbulldog`) in the filename, but since there's no set format for filenames it's likely to fail to find what you're looking for 😅

**Comment by thickconfusion** [comment]
Would it be possible to also look also at any title which has been applied
to the Scene?

**Comment by Maista6969** [comment]
It will try the title or URL, whichever is set: if there's a title it will use that, but once again it's just going to be a best effort guess: what do your titles look like?

**Comment by thickconfusion** [comment]
I installed the ScrapeWithURL and also started adding some of those. I tried the button next to URL, and also Scrape with> RedGifs. Now getting this error:

```
ERRO[2024-05-03 15:45:33] could not unmarshal json from script output: json: cannot unmarshal object into Go value of type []scraper.ScrapedScene
ERRO[2024-05-03 15:45:33] scrapeSingleScene: input: scrapeSingleScene error while name scraping with scraper Redgifs: could not unmarshal json from script output: json: cannot unmarshal object into Go value of type []scraper.ScrapedScene
2024-05-03T15:45:33.959341383Z INF Response: 200 OK httpRequest={"proto":"HTTP/1.1","remoteIP":"192.168.1.209:55684","requestID":"4732f1db2dbe/CbLdzuXxeo-001386","requestMethod":"POST","requestPath":"/graphql","requestURL":"http://<<>>.lan:9999/graphql"} httpResponse={"bytes":184,"elapsed":299.660428,"status":200} service=stash
ERRO[2024-05-03 15:45:41] could not unmarshal json from script output: json: cannot unmarshal object into Go value of type []scraper.ScrapedScene
ERRO[2024-05-03 15:45:41] scrapeSingleScene: input: scrapeSingleScene error while name scraping with scraper Redgifs: could not unmarshal json from script output: json: cannot unmarshal object into Go value of type []scraper.ScrapedScene

```

**Comment by Maista6969** [comment]
I apologize for that, somehow it slipped through my testing: fixed in version `2e4e19c`

**Comment by thickconfusion** [comment]
This works. @Maista6969 Can you please delete this issue based on privacy?
